docs/question.md

```
@@ -374,10 +364,35 @@ int m = 4;
+
+<!-- prettier-ignore -->
+```html title="question.html"
+<markdown>
+  $$\phi = \frac{1+\sqrt{5}}{2}$$
+
+  Need to use `$` outside of math-mode? Place inside inline code.
+  So, excel formulas would look like: `$A$1`, `B$5`, `$C9`, `D2`.
+</markdown>
+```
+
+![](images/latex.png)
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)